Package.swift

```diff
@@ -1,13 +1,13 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 import PackageDescription
 
 let package = Package(
     name: "Charts",
     platforms: [
-          .iOS(.v9),
-          .tvOS(.v9),
-          .macOS(.v10_11),
+        .iOS("13.0"),
```

Comment:
```suggestion
        .iOS("14.0"),
```
Optional: we could consider 14.0 if desired, due to bumping our minimum version recently (same for tvOS).